library/src/main/java/org/pattonvillerobotics/commoncode/robotclasses/opencv/ImageProcessor.java

```diff
@@ -67,25 +64,28 @@ public void onManagerConnected(int status) {
             }
         }
 
-        while (!initialized) {
-            try {
-                Thread.sleep(50);
-            } catch (InterruptedException e) {
-                e.printStackTrace();
-            }
+        while (!initialized && opMode.opModeIsActive()) {
```

Comment:
`opModeIsActive()` calls `idle()` internally

Reply:
Also, you should investigate using `wait()` and `notify()` instead of a busy loop.